### ISSUE TITLE
Baby weight and baby temp fields should be decimals, not integers

### ISSUE THREAD

**Issue by cafootitt** [OpenSRP/opensrp-client-chw-anc#304]
In the PNC home visit, PNC health facility visit task, the two fields: `baby_weight` and `baby_temp` should be decimal inputs, not integers.

To pass QA:
- [x] I should be able to enter any decimal weight for the baby between > 0 and < 20
- [x] I should be able to enter any decimal temp for the baby between >= 35 and <= 42
- [x] Works in both English and French versions

**Comment by cafootitt** [comment]
V1.0.6

This is not working yet. I can't tap on the decimal button. it doesn't get recorded.
This is an urgent priority to fix on Friday, Oct 4 before the app update goes out.

**Comment by cafootitt** [comment]
QA V1.0.6

I'm witnessing some inconsistent behavior here. When I have the app installed on my device, then open a PNC home visit, I cannot record decimals for these two fields. However, when I moved the app to Samsung Game Launcher, so that I could do a screen recording to demonstrate the issue, the decimals started working!! Then, when I removed the app from Game Launcher, the decimals stopped working again!!!

Does anyone know how we can explain this or what we can do to make it work?
